src/video.tsx

```diff
@@ -94,12 +97,20 @@ export class Video extends Component<VideoProps, VideoComponentState> {
   }
 
   static defaultProps = {
+    defaultTime: 0,
     sourceType: 'video',
     autoPlay: false,
     controls: false,
     preload: isSafari ? 'auto' : 'metadata'
   }
 
+  componentDidMount() {
+    const { defaultTime } = this.props;
+    if (this.currentElement) {
+      this.currentElement.currentTime = defaultTime;
+    }
+  }
+
   componentDidUpdate(prevProps: VideoProps) {
```

Comment:
What if consumer changes the defaultTime prop? Do we want to react to that?

Reply:
how other components behave in that case? I don't think so. But I am not sure. Let's think - the only think I can do is to override current state, which is silly imo